config/custom-environment-variables.json

```diff
@@ -38,5 +38,9 @@
       "enabled": "EVIDENCE_UPLOAD_ENABLED",
       "maxFileSize": "MAX_FILE_SIZE"
     }
+  },
+  "postcodeChecker": {
+    "url": "POSTCODE_CHECKER_URL",
+    "token": "POSTCODE_CHECKER_TOKEN"
```

Comment:
shouldn't this also include `allowedCountries`?

Reply:
Not going to make allowedCountries config as we can only add Scotland to this list and if we do that we might as well remove the check.